cartaya/card-3/resources/css/style.css

```diff
@@ -15,40 +15,43 @@ body {
     margin: 0;
 }
 
-.card-3 {
+.camphor-font {
+    font-family: 'Camphor-Light', 'Arial', sans-serif;
+    font-size: 16px;
+    font-weight: 500;
+    letter-spacing: 0.2px;
+    text-align: center;
+}
+
+.card {
     align-items: center;
     border-radius: 4px;
+    color: rgb(41, 41, 41);
+    color: rgba(0, 0, 0, 0.54);
     display: flex;
     flex-flow: column;
```

Comment:
Why you use the flex-flow property in this case? The flex-flow property is a shorthand property for the flex-direction and the flex-wrap properties.